app/helpers/review_mapping_helper.rb

```diff
@@ -97,7 +97,7 @@ def sort_reviewer_by_review_volume_desc
       r.avg_vol_in_round_2,
       r.avg_vol_in_round_3 = Response.get_volume_of_review_comments(@assignment.id, r.id)
     end
-    @reviewers.sort! {|r1, r2| r2.overall_avg_vol <=> r1.overall_avg_vol }
+    # @reviewers.sort! {|r1, r2| r2.overall_avg_vol <=> r1.overall_avg_vol }
```

Comment:
This function was broken when we got the code and caused the search bar on the related UI page to throw an error.  As it was just a sort of the results, I felt it was ok to remove to fix the overall functionality.